src/components/button/button.tsx

```diff
@@ -5,7 +5,7 @@ import styles from './button.module.scss';
 type Props = {
   children: ReactNode;
   className?: string;
-  type?: 'primary' | 'secondary' | 'link' | 'text';
+  type?: 'primary' | 'secondary' | 'link' | 'text' | 'text-link';
```

Comment:
Why not just use type `text`? 

If you dislike the underline, let's override it using component style instead of adding another "generic" button type.

Reply:
Yeah I actually had it like this before. I reverted this to rather override the button styling. Needed a new optional prop though - `btnClassName`